app/src/main/java/com/smithnoff/mytaskyapp/utils/SessionManagerUtil.kt

```diff
@@ -0,0 +1,35 @@
+package com.smithnoff.mytaskyapp.utils
+
+import android.content.SharedPreferences
+import com.smithnoff.mytaskyapp.data.models.UserLoginResponse
+import javax.inject.Inject
+import javax.inject.Singleton
+
+const val USER_FULLNAME = "name"
+const val USER_ID = "id"
+const val USER_TOKEN = "token"
+
+@Singleton
+class SessionManagerUtil @Inject constructor(private val sharedPreferences: SharedPreferences) {
+
+    fun startUserSession(user: UserLoginResponse) {
+        with(sharedPreferences.edit()) {
+            putString(USER_FULLNAME, user.fullName)
+            putString(USER_ID, user.userId)
+            putString(USER_TOKEN, user.token)
+            apply()
+        }
+    }
+
+    fun getSessionInfo() = UserLoginResponse(
+        sharedPreferences.getString(USER_FULLNAME, "")!!,
+        sharedPreferences.getString(USER_ID, "")!!,
+        sharedPreferences.getString(USER_TOKEN, "")!!
+    )
```

Comment:
In general, I'm a big fan of always using named parameters when you can. In PRs nobody will be able to tell what a parameter stands for since there are no inline hints. In this case it's obvious of course, but I personally feel like named params add a lot of readability regardless